### DISCUSSION TITLE
How to toggle comppnent 'locked' or any other property programmatically?

### DISCUSSION THREAD

**Discussion by AbdallahAbdedaiem** [GrapesJS/grapesjs#4945] [Q&A]
I want to set locked property initially to true, then unlock the component on certain condition. I tried some syntax for that as for exp:
`gjsComponent.set('locked', isLocked);
` but nothing seems to work

**Comment by artf** [comment]
That should work and it seems to work properly on my side/demo examples, so please provide a reproducible demo of you use case.
